fawkes/configs/app_config.py

```diff
@@ -266,6 +267,29 @@ def __init__(self, config):
         self.oauth_params = config["oauth_params"]
         self.query_list = config["query_list"]
 
+class SplunkReviewChannel(ReviewChannel):
+    """ The configurations specific to Splunk.
+
+    Attributes:
+        host: splunk host for api.
+        port: port for api.
+        username: username to authenticate.
+        password: password to authenticate.
+
+    """
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.host = config["host"]
+        self.port = config["port"]
+        self.username = config["username"]
+        self.password = config["password"]
+        self.query = config["query"]
+        self.earliest = config["earliest_time"]
+        self.latest = config['latest_time']
```

Comment:
Can't this be a part if the query itself ?

Reply:
Updated the query to contain earliest and latest. also referenced the Splunk documentation for the same